version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2015.06.19-1-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.0.1"
```

Comment:
Yes, because the date versioning was provided by the plugin I removed - and since Michael switched us to semantic versioning by default on other projects, it seemed reasonable to do here as well.